packages/client/hmi-client/src/workflow/ops/optimize-ciemss/tera-optimize-ciemss.vue

```diff
@@ -494,6 +494,14 @@ const runOptimize = async () => {
 
 	const simulationResponse = await makeForecastJobCiemss(simulationPayload);
 	getStatus(simulationResponse.id);
+	const state = _.cloneDeep(props.node.state);
+	emit('append-output', {
+		type: OptimizeCiemssOperation.outputs[0].type,
+		label: `Simulation output - ${props.node.outputs.length + 1}`,
+		value: simulationResponse.id,
+		isSelected: false,
+		state
+	});
```

Comment:
I think `value` needs to be wrapped in an array`value: [simulationResponse.id]`

Reply:
Youre right, i will change this in ciemss node too